syspurpose/man/syspurpose.8

```diff
@@ -77,7 +77,7 @@ foo bar\fP is identical to \fBsyspurpose remove-addons foo bar\fP.
 .TP
 \fBsyspurpose unset \fIPROPERTY\fP \fIVALUE\fP\fP
 Clear set addons. E.g. \fBsyspurpose unset addons
-foo bar\fP is identical to \fBsyspurpose unset-addons foo bar\fP.
+\fP is identical to \fBsyspurpose unset-addons\fP.
```

Comment:
Can you please change the text little bit. It looks like that `\fP` at the beginning of the line add there line break. I would like to have explanation on one line:

```
Clear set addons. E.g. \fBsyspurpose unset addons\fP is identical to \fBsyspurpose unset-addons\fP.
```